package.json

```diff
@@ -9,6 +9,6 @@
   "dependencies": {
     "JSONStream": "^1.3.5",
     "ws": "^7.3.0",
-    "redis": "^3.0"
```

Comment:
Why is this dependency removed? It is required for the operation of the program subsequently.